src/domain/session/SessionViewModel.js

```diff
@@ -81,12 +81,12 @@ export class SessionViewModel extends ViewModel {
         }));
         this._updateCreateRoom(createRoom.get());
 
-        const lightbox = this.navigation.observe("lightbox");
-        this.track(lightbox.subscribe(eventId => {
-            this._updateLightbox(eventId);
-        }));
-        this._updateLightbox(lightbox.get());
-
+        setupLightboxNavigation(this, 'lightboxViewModel', (eventId) => {
+            return {
+                room,
+                eventId,
+            };
+        });
```

Comment:
Replacing all of the previous boilerplate to setup the lightbox with this composable function that external consumers can use in the SDK.

This makes setting up the lightbox in your project that uses the SDK as simple as this `setupLightboxNavigation(...)` function call.